docs/docs/guides/basics/eth.md

```diff
@@ -344,6 +343,358 @@ Transaction receipt: {
+    //   transactionHash: '0x0ffe880776f5631e4b64caf521bd01cd816dd2cc29e533bc56f392211856cf9a',
+    //   transactionIndex: 0n,
+    //   type: 0n
+    // }
+}
+(async () => {
+    await test();
+})();
+```
+
+### EIP-1559
+Ethereum Improvement Proposal 1559 is a significant upgrade to the Ethereum network's fee market and transaction pricing mechanism. It was implemented as part of the Ethereum London hard fork, which occurred in August 2021. EIP-1559 introduces several changes to how transaction fees work on the Ethereum blockchain, with the primary goals of improving user experience and network efficiency.
+
+Here are some of the key features and changes introduced by EIP-1559:
+
+1. Base Fee: EIP-1559 introduces a concept called the "base fee." The base fee is the minimum fee required for a transaction to be included in a block. It is determined algorithmically by the network and adjusts dynamically based on network congestion. When the network is busy, the base fee increases, and when it's less congested, the base fee decreases.
+
+2. Inclusion Fee: In addition to the base fee, users can voluntarily include a "tip" or "inclusion fee" to incentivize miners to include their transactions in the next block. This allows users to expedite their transactions by offering a tip to miners.
+
+3. Predictable Fees: With EIP-1559, users have a more predictable way to estimate transaction fees. They can set the maximum fee they are willing to pay, which includes the base fee and the tip. This eliminates the need for users to guess the appropriate gas price.
+
+4. Burn Mechanism: EIP-1559 introduces a mechanism by which the base fee is "burned" from circulation, reducing the overall supply of Ether (ETH). This deflationary mechanism can help address some of the concerns related to the increasing supply of ETH and potentially make it a better store of value.
+
+5. Improved Fee Auctions: Under EIP-1559, fee auctions are more efficient. Users specify the maximum fee they are willing to pay, and the protocol automatically adjusts the tip to ensure transactions get processed promptly without overpaying.
+
+6. Simpler Transaction Process: Users experience a simplified transaction process, as they don't have to set gas prices manually. Instead, they specify the maximum fee they are willing to pay, and the wallet software handles the rest.
+
+EIP-1559 has been well-received for its potential to create a more user-friendly and efficient transaction fee system, making the Ethereum network more accessible and predictable for users. It is also seen as an important step in the transition to Ethereum 2.0, which aims to address scalability and sustainability challenges on the network.
+
+To send EIP-1559 transaction use code below:
+
+```typescript
+import Web3 from 'web3';
+const web3 = new Web3('http://localhost:8545');
+
+async function test() {
+    const privateKey = '0x1f953dc9b6437fb94fcafa5dabe3faa0c34315b954dd66f41bf53273339c6d26';
```

Comment:
in all code examples remove hardcoded private key, and add some dummy env variable for it